src/platform/plugins/shared/visualizations/public/visualize_app/utils/get_table_columns.tsx

```diff
@@ -11,6 +11,7 @@ import React from 'react';
 import { EuiBetaBadge, EuiButton, EuiEmptyPrompt, EuiIcon, EuiBadge } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
 import { FormattedMessage } from '@kbn/i18n-react';
+import { CustomSortingOptions } from '@kbn/content-management-table-list-view-table/src/components';
```

Comment:
it should be `import type { CustomSortingOptions } from '@kbn/content-management-table-list-view-table'`